lib/content/content.go

```diff
@@ -15,11 +16,13 @@ type Content struct {
 
 type provider struct {
 	alias string
-	meta  interface{}
+	meta  map[string]interface{}
 }
 
 var providers []provider
 
+var DEFAULT_PRIORITY float64 = 10.0
```

Comment:
You do not have to rewrite, right?
I think it should be using const.

Reply:
Oh...You are right.
Fixed 9b09840e736bdaf0b13ea2dcd061ab9d86b36863